ios/browser/skus/skus_sdk_service_factory.cc

```diff
@@ -0,0 +1,60 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/ios/browser/skus/skus_sdk_service_factory.h"
+
+#include "brave/components/skus/browser/skus_sdk_service.h"
+#include "components/keyed_service/core/keyed_service.h"
+#include "components/keyed_service/ios/browser_state_dependency_manager.h"
+#include "ios/chrome/browser/browser_state/browser_state_otr_helper.h"
+#include "ios/chrome/browser/browser_state/chrome_browser_state.h"
+#include "ios/web/public/browser_state.h"
+#include "services/network/public/cpp/shared_url_loader_factory.h"
+
+namespace skus {
+
+// static
+mojom::SkusSdk* SkusSdkServiceFactory::GetForBrowserState(
+    ChromeBrowserState* browser_state) {
+  return nullptr; // Currently don't have the correct type to return
```

Comment:
iOS still does not have a mojo implementation it can use as the one found in the component uses desktop only devs (`content`)

Reply:
See original comment: https://github.com/brave/brave-core/pull/10847/files#r751825325

Reply:
Fixed with https://github.com/brave/brave-core/pull/10847/commits/79225d0b248112b48414fe358dd4c6063815366d